omise-woocommerce.php

```diff
@@ -59,10 +59,10 @@ public function __construct()
 	public function embedded_form_notice()
 	{
 		$this->omiseCardGateway = new Omise_Payment_Creditcard();
-		$embedded_form_enabled = $this->omiseCardGateway->get_option('embedded_form_enabled');
+		$secure_form_enabled = $this->omiseCardGateway->get_option('secure_form_enabled');
```

Comment:
Let's find a way to move beyond [stringly typed ](https://wiki.c2.com/?StringlyTyped) interfaces